src/filing/utils/date.js

```diff
@@ -68,4 +69,29 @@ export const isDstObserved = (date = new Date()) => {
 export const easternOffsetHours = (date = new Date()) => {
   const eastOffset = isDstObserved() ? 4 : 5
   return date.getTimezoneOffset() / 60 - eastOffset
-}
\ No newline at end of file
+}
+
+/**
+ * Function to format timestamp and return a string with eastern timezone
+ * @param {Number} timeToFormat timestamp of when receipt was submitted
+ * @returns formatted string that includes month, day, year and time from ET
+ */
+export const formatReceiptTime = timeToFormat => {
+  let timezone = {
+    timeZone: 'America/New_York',
+  }
+
+  const options = {
+    year: 'numeric',
+    month: 'long',
+    day: 'numeric',
+    timezone,
```

Comment:
This probably needs to be spread.  As you currently have it, you would end up with a nested object at the key `timezone`.
```suggestion
    ...timezone,
```

Reply:
You are correct. Code has been updated.